icu4j/main/classes/core/src/com/ibm/icu/message2/functions/DateTimeFormatterFactory.java

```diff
@@ -0,0 +1,120 @@
+// © 2022 and later: Unicode, Inc. and others.
+// License & terms of use: http://www.unicode.org/copyright.html
+
+package com.ibm.icu.message2.functions;
```

Comment:
The reason I did it was for modularity.

And to enforce that the standard functions really built on top of public APIs
(because they can't access package APIs).
So it "proves" that one can replace them (with ones implemented on top of Android APIs, for example)

And they are public because they are in a different package.

---

I moved them.

Reply:
Done.